joomla/joomla-cms
# [5.0] SQL fix and update scripts for PR #41259

Follow-up Pull Request (PR) for PR #41259 .

### Summary of Changes

Add update SQL scripts for migrating the filter parameters of relevant core field plugins and the fieldparams of relevant custom fields on update.

### Testing Instructions

Have a clean 5.0.0-alpha3 or previous 5.0.0 alpha version or a 4.4-alpha3.

Create a few custom fields of types "text", "textarea" or "editor" for articles, and for some of these fields use the "Text" filter, for some use the filter from the plugin and for others use some other filter.

Now update to a new package build on the 5.0-dev branch for the actual result, or update to the patched package or custom update URL created by Drone for this PR for the expected result.

Then check the filter parameter of the fields plugins in the `#__extensions` table.

Then check the filter parameter of the field's fieldparams plugins in the `#__fields` table.

Finally create an article using the custom fields created before the update and save that article.

### Actual result BEFORE applying this Pull Request

After an update, the parameters and fieldparams have not been migrated.

### Expected result AFTER applying this Pull Request

After an update, the parameters and fieldparams have been migrated.

Saving the article with the custom fields works.

### Link to documentations
Please select:
